quicklogic_fasm/convert_csv_to_db.py

```diff
@@ -40,8 +40,83 @@ class QLDbEntry(DbEntry):
         'macro_interface_top_right': 'INTERFACE'
     }
 
+    # A map of grid row coordinates to bit WL ranges.
+    # Taken from QL732B_cdl.cmd
+    wl_map = {
+        1: (1, 28,),
+        2: (29, 56,),
+        3: (57, 84,),
+        4: (85, 112,),
+        5: (113, 140,),
+        6: (141, 168,),
+        7: (169, 196,),
+        8: (197, 224,),
+        9: (225, 252,),
+        10: (253, 281,),
+        11: (282, 309,),
+        12: (310, 337,),
+        13: (338, 365,),
+        14: (366, 393,),
+        15: (394, 421,),
+        16: (422, 449,),
+        17: (450, 477,),
+        18: (478, 505,),
+        19: (506, 533,),
+        20: (534, 561,),
+        21: (562, 589,),
+        22: (590, 617,),
+        23: (618, 645,),
+        24: (646, 674,),
+        25: (675, 702,),
+        26: (703, 730,),
+        27: (731, 758,),
+        28: (759, 786,),
+        29: (787, 814,),
+        30: (815, 842,),
+    }
+
+    # A map of grid col coortinates to bit BL ranges
+    # Taken from QL732B_cdl.cmd
+    bl_map = {
```

Comment:
are those maps always constant? e.g. does it scale to pp3e? If not please file an issue about it (we can merge the constans now, but we need to know this is a workaround)

Reply:
I think that is not a workaround and these maps are identical for both EOS-S3 and PP3E. I assume that both FPGAs are referred to as `QL732B` when it comes to bitstream.

I derived these maps from the `QL732B_cdl.cmd` file which is not provided by QuickLogic in their EOS-S3 and PP3E repositories.

I also got the information:
> In file, ColClk_ql732b.csv, second column corresponds to WL_bit and third column corresponds to BL_bit. Then in file, QL732B_cdl.cmd, look at define_row & define_column to see which row & col it belongs.